components/shortcuts/index.js

```diff
@@ -8,19 +8,19 @@ const ITEMS = [
   {
     title: 'Interchain',
     description: '',
-    url: '/interchain',
```

Comment:
can we route these to a default page for that section?

Reply:
There isn't a default page for each section, so I linked it to the section's first page.